release template/CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at opensource@github.com. All
+reported by contacting the project team at opensource@<COMPANY_NAME>.com. All
```

Comment:
maybe more generic too, since it might get wonky with a find/replace for `<COMPANY_NAM>` if the value is something like "Jane's Hats". Maybe a new placeholder like `<OPEN_SOURCE_MAILBOX>`

Reply:
Great suggestion! I've pushed 34409b9 to put in the new placeholder.